File: histogram-v1/histogramV1.js
const histogram = (array) => {

    let output = "";
    let num = 6;

    [...array].reverse().forEach((element) => {
        output += num + "|";
        num--;
        if(element === 0) {
            output += "\n";
        } else {
            output += "#".repeat(element) + ` ${element}` + "\n";
        };
    });

    return output;

};

module.exports = { histogram };

//https://www.codewars.com/kata/57d532d2164a67cded0001c7/train/javascript

File: python-venv/histogram_v1/kata_function/histogram_v1.py
def histogram(array):

    output = ""
    dice = 6
    rev = array[::-1]

    for i in range(len(rev)):
        output += str(dice) + "|"
        dice -= 1
        if rev[i] == 0:
            output += "\n"
        else:
            output += "#" * rev[i] + " " + str(rev[i]) + "\n"
            
    return output

#https://www.codewars.com/kata/57d532d2164a67cded0001c7/train/python